package.json

```diff
@@ -111,6 +111,7 @@
     "buffer": "^6.0.3",
     "lodash": "^4.17.21",
     "lodash.orderby": "^4.6.0",
+    "lottie-react": "^2.3.1",
```

Comment:
is this just being used for just the loader? can we do a standard CSS animation instead of adding a dependency for it?

Reply:
Yes, it for the loader. Lottie is very lightweight, and this animation was asked for in the designs. It could be changed to a css animation in the future, but that would need some time to figure out how to do it